ui/apps/platform/src/Components/NotFoundMessage/NotFoundMessage.tsx

```diff
@@ -34,17 +32,12 @@ const NotFoundMessage = ({
                 <EmptyStateHeader titleText={title} headingLevel="h1" />
                 <EmptyStateFooter>
                     {message && <EmptyStateBody>{message}</EmptyStateBody>}
-                    {isButtonVisible && <Button variant="primary">{actionText}</Button>}
-                    {isLinkVisible && (
-                        <Button
-                            variant={ButtonVariant.link}
-                            isInline
-                            component={LinkShim}
-                            href={url}
-                        >
+                    {isButtonVisible && (
+                        <Button variant="primary" onClick={onClick}>
```

Comment:
The `onClick` wasn't even being used before?

Reply:
Should have been, but in only one (out of the way) place.